### PR TITLE
Add info about omitting the <ending-shape> keyword

### PR DESCRIPTION
**Pull request by bytrangle**
### Description

When rendering radial gradient, it's possible to omit the `<ending-shape>` keyword and the browser is still able to render the correct shape if the right size values are given:
- If there is only one value in `<length>` unit, the gradient shape is provided as a circle.
- If there are two values in `<length-percentage>` unit, the gradient shape is provided as an ellipse.

### Motivation

I think it's not obvious from the page on radial gradient that it's possible to omit `circle` or `ellipse` keyword. The current wording and subsequent examples makes it sound like users have to include both the shape and the corresponding sizes to be valid.

### CODE REVIEW

File: files/en-us/web/css/gradient/radial-gradient/index.md [hunk 1]
```diff
@@ -52,7 +52,9 @@ A radial gradient is specified by indicating the center of the gradient (where t
 
     If `<ending-shape>` is specified as `circle`, the size may be given explicitly as a [`<length>`](/en-US/docs/Web/CSS/length), which provides an explicit circle radius. Negative values are invalid.
 
-    If `<ending-shape>` is specified as `ellipse` or is omitted, the size may be given as a [`<length-percentage>`](/en-US/docs/Web/CSS/length-percentage) with two values to provide an explicit ellipse size. The first value represents the horizontal radius, the second the vertical radius. Percentages values are relative to the corresponding dimension of the gradient box. Negative values are invalid.
+    If `<ending-shape>` is specified as `ellipse`, the size may be given as a [`<length-percentage>`](/en-US/docs/Web/CSS/length-percentage) with two values to provide an explicit ellipse size. The first value represents the horizontal radius, the second the vertical radius. Percentages values are relative to the corresponding dimension of the gradient box. Negative values are invalid.
```
Comment (estelle) [inline files/en-us/web/css/gradient/radial-gradient/index.md:55]:
```suggestion
    If `<ending-shape>` is specified as `ellipse`, the size may be given as a [`<length-percentage>`](/en-US/docs/Web/CSS/length-percentage) with two values to provide an explicit ellipse size. The first value represents the horizontal radius and the second is the vertical radius. Percentage values are relative to the corresponding dimension of the gradient box. Negative values are invalid.
```

File: files/en-us/web/css/gradient/radial-gradient/index.md [hunk 2]
```diff
@@ -52,7 +52,9 @@ A radial gradient is specified by indicating the center of the gradient (where t
...
+
+    The `<ending-shape>` keyword may be omitted and the gradient shape will be determined based on the size given. One `<length>` value provides a circle, while two values in `<length-percentage>`units provide an ellipse.
```
Comment (estelle) [inline files/en-us/web/css/gradient/radial-gradient/index.md:57]:
```suggestion
    When the `<ending-shape>` keyword is omitted, the gradient shape is determined by the size given. One `<length>` value provides a circle, while one `<percentage>` value or two values in `<length-percentage>`units provide an ellipse.
```

I think if the value is a percentage, then it matches the container, therefore an ellipse.

**Review by estelle — COMMENTED** [review]
A few suggestions, and some grammar edits on the original text.

File: files/en-us/web/css/gradient/radial-gradient/index.md (hunk 2 above)
Comment (bytrangle) [reply files/en-us/web/css/gradient/radial-gradient/index.md:57]: Hi Estelle. Thanks for many valuable suggestions. However, it seems like giving only one `<percentage>` value is not invalid.

[Demo](https://jsfiddle.net/r9kdbLgc/)

Comment (estelle) [reply files/en-us/web/css/gradient/radial-gradient/index.md:57]: Thanks! I think your suggested changes are really helpful. I added a sentence to my original suggestion to let devs know a percent alone is not valid.

Comment (bytrangle) [reply files/en-us/web/css/gradient/radial-gradient/index.md]: Can we mark it as resolved now?

**Review by estelle — APPROVED** [review]
Thanks!

**Pull request merged**